developers/vscode/README.md

```diff
@@ -0,0 +1,39 @@
+# Ocean Protocol VSCode Extension
+
+The Ocean Protocol VSCode extension enables you to run compute jobs seamlessly within Visual Studio Code using Ocean Protocol's secure compute-to-data network. Execute your algorithms (JavaScript or Python) on data without ever exposing the raw data.
+
+## Getting Started
+
+Once installed, the extension adds an Ocean Protocol section to your VSCode workspace. Here you can configure your compute settings and run compute jobs using the currently active algorithm file.
+
+### Optional Setup
+
+Feel free to use the default setup options or you also have the option to customise your compute settings:
+1. **RPC URL** – Your blockchain connection endpoint.
+2. **Ocean Node URL** – The node managing compute jobs.
+3. **Private Key** – For signing transactions (stored securely).
```

Comment:
I didn't know that we have default value for private key, I do not think it should be placed as optional setup.

Reply:
Yeah, it's optional. You don't need to input a private key to start a free compute job.

Reply:
can we mention this please?